### PR TITLE
new provider: nzbindex.nl

### PR DESCRIPTION
**Pull request by nielsw**
Hi,

i've added nzbindex.nl as a search provider, would you mind taking a look at it and/or even adding it as an optional search provider?

Thanks for all your work so far / the time you may take looking at this.

Kind Regards,

Niels Wijnja

### CODE REVIEW

**Comment by mizzos** [comment]
+1 on this.

**Comment by basrieter** [comment]
+1 on this one to. We need more free search providers.

**Comment by pamarama** [comment]
+1 for me too

File: sickbeard/providers/nzbindex_nl.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+		results = []
+
+		for curItem in items:
+			(title, url) = self._get_title_and_url(curItem)
+
+			if not title or not url:
+				logger.log(u"The XML returned from the NZBindex.nl RSS feed is incomplete, this result is unusable: "+data, logger.ERROR)
+				continue
+			if not title == 'Not_Valid':
```
Comment (midgetspy) [inline sickbeard/providers/nzbindex_nl.py]: Should be != for clarity

File: sickbeard/providers/nzbindex_nl.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+			if title.lower().count(showName.lower()) == 1:
+				titleStart = title.lower().index(showName.lower())
+				titleEnd = title.index(" ",titleStart)
+				title = title[titleStart:titleEnd]
+			elif title.lower().count(showName.lower()) > 1:
+				titleStart = title.lower().rindex(showName.lower())
+				titleEnd = title.index(" ",titleStart)
+				title = title[titleStart:titleEnd]
+			else:
+				title = "Not_Valid"
```
Comment (midgetspy) [inline sickbeard/providers/nzbindex_nl.py]: I'd just use None for this rather than "Not_Valid"

**Comment by schumi2004** [comment]
Why not just nzbindex.com instead of *.nl ?

**Comment by ghost** [comment]
Why not just nzbindex.nl instead of *.com ?

Try "nzbindex" as keyword in Google, that will give you your answer.

**Comment by thezoggy** [comment]
.com = us gov control and can easily be shutdown per ICE.

File: sickbeard/providers/nzbindex_nl.py (hunk 1 above)
Comment (nielsw) [inline sickbeard/providers/nzbindex_nl.py]: Fair enough, was a leftover from 'not title', will be changed.

File: sickbeard/providers/nzbindex_nl.py (hunk 2 above)
Comment (nielsw) [inline sickbeard/providers/nzbindex_nl.py]: Again a leftover, will change it.

**Comment by nielsw** [comment]
The name of the searchprovider isn't really relevant imho, we could change it to just 'Nzbindex' to avoid confusion but in the meantime it'll be .nl as it's hosted in the Netherlands.

**Comment by midgetspy** [comment]
OK I tested this and I'm not going to be pulling it. This is not an index it's simply a usenet listing. An index filters and categorizes the content so that we can search it, this site doesn't. Usenet indexes have complex sets of regexes to decipher the release title reliable - we would need this too to make this provider useful. I tried searching for an ep of a popular show and it couldn't find any because it was parsing the titles wrong - and to parse them right is an indexers job, not SB's.

**Pull request closed without merging**